src/ucp/api/ucp.h

```diff
@@ -418,12 +418,16 @@ enum ucp_am_cb_flags {
  * @brief Flags for sending a UCP Active Message.
  *
  * Flags dictate the behavior of ucp_am_send_nb
- * currently the only flag tells UCP to pass in
+ * UCP_AM_SEND_REPLY tells UCP to pass in
```

Comment:
pls document each flag behavior near the flag definition, see other examples of flags doumentation in UCP

Reply:
unhandled